adbms/src/test/scala/de/up/hpi/informationsystems/adbms/function/SequentialFunctorTest.scala

```diff
@@ -102,6 +109,88 @@ class SequentialFunctorTest extends TestKit(ActorSystem("sequential-functor-test
         val recipients = Seq(partnerDactor1, partnerDactor1, partnerDactor1)
         testSimpleSeqFunctor(recipients, "same receiver multiple times")
       }
+
+    }
+
+    "using a functor context" should {
+      val probe = TestProbe()
+      val startMessage = MessageA.Request("first message")
+
+      def testSeqFunctorWithContext(testFunction: FunctorContext[MessageA.Request] => Unit): Unit = {
+        implicit val sender: ActorRef = probe.ref
+
+        val fut = SequentialFunctor()
+          .start((a: MessageA.Request) => a, Seq(partnerDactor1))
+
```

Comment:
Aren't blank lines between call-chains like really bad style? It caught me off guard at least.